restapi/src/main/java/io/stargate/web/docsapi/service/ReactiveDocumentService.java

```diff
@@ -305,9 +344,11 @@ public ReactiveDocumentService(
       String collection,
       String documentId,
       String payload,
-      ExecutionContext context) {
+      String ttl,
```

Comment:
please no string! Here integer is fine, this is where we allow the ttl to be specified..